templates/localstorageclass.yaml

```diff
@@ -9,6 +9,8 @@ metadata:
 spec:
   kubernetes:
     minSupportedVersion: v1.14.0
+  cloudProvider:
```

Comment:
think it's safe to delete this addon.

Reply:
Perhaps so, but that would fall outside of the domain of this PR.